packages/types/src/interfaces/types.ts

```diff
@@ -1,5 +1,26 @@
 // Auto-generated via `yarn polkadot-types-from-defs`, do not edit
 /* eslint-disable */
+import { AccountId, Address } from '@polkadot/types/interfaces';
+import { AnyNumber } from '@polkadot/types/types';
+import BN from 'bn.js';
+
+export {
```

Comment:
Need to export these types from '@polkadot/types/interfaces' so that we can use it from cennznet/types.
The reason we cannot export all i.e export * from '@polkadot/types/interfaces'  is because there is overlap of several types..